frontend/src/components/common/DateInput/index.tsx

```diff
@@ -33,7 +33,6 @@ export default function DateInput({ label, innerText, ...props }: DateInputProps
         outline={false}
         size="sm"
         color="white"
```

Comment:
`button` 요소의 `type`를 `button`으로 지정해주면 의도치 않은 이벤트 발생으로부터 조금 더 안전하지 않을까 싶습니다.

```suggestion
        outline={false}
        size="sm"
        color="white"
        type="button"
```

Reply:
중요한 부분 짚어주셨네용!! 감사합니다~